tests/unit/drivers/test_rankingevaluation_driver.py

```diff
@@ -21,43 +22,51 @@ def evaluate(self, matches_ids, groundtruth_ids, *args, **kwargs) -> float:
 
```

Comment:
How about converting `groundtruth_ids` to `set` for faster lookup? It can be useful for high `eval_at`

Reply:
it is important to keep the order which the set does not guarantee I think